package.json

```diff
@@ -76,9 +76,9 @@
     "react-transition-group": "^4.4.2",
     "remark-parse": "^7.0.1",
     "shopify-buy": "^2.6.1",
-    "slate": "^0.58.4",
-    "slate-history": "^0.58.4",
-    "slate-react": "^0.58.4",
+    "slate": "^0.81.1",
```

Comment:
I don't know which thing causes it but entering 0 (or any number) scrolls the page to some amount until the input area is approximately at the center of the window ?_?

Reply:
For some inexplicable reason, slate appears to be updating the scroll position when the cursor selection is updated ಠ_ಠ
...and for some reason numbers make it offset more than letters ಠ_ಠ (you can see the scrolling if `position: sticky` is removed)